GiTinder/Models/Responses/LanguagesResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GiTinder.Models.Responses
+{
+    public class LanguagesResponse
+    {
+    }
+}
```

Comment:
@Riceqrisp this is not needed

Reply:
Won't be needed for get languages endpoint? :) removed.